ucp/_libs/ucx_api_dep.pxd

```diff
@@ -103,8 +103,6 @@ cdef extern from "ucp/api/ucp.h":
     int UCP_EP_PARAMS_FLAGS_NO_LOOPBACK
     int UCP_EP_PARAMS_FLAGS_CLIENT_SERVER
 
-    ctypedef ucp_conn_request* ucp_conn_request_h
```

Comment:
Nice catch!

Reply:
Yeah, the verbose flag really helps catching Cython errors when building, e.g. `pip install -v -e .`